src/tye/ProjectEvaluation.targets

```diff
@@ -1,15 +1,21 @@
 ﻿<Project>
-    <Target Name="MicrosoftTye_GetProjectMetadata" DependsOnTargets="Restore;ResolveReferences;ResolvePackageDependenciesDesignTime;PrepareResources;GetAssemblyAttributes" >
+    <PropertyGroup>
+        <MicrosoftTye_GetProjectMetadata_DependsOn>Restore;ResolveReferences;ResolvePackageDependenciesDesignTime;PrepareResources;PrepareResources;GetAssemblyAttributes</MicrosoftTye_GetProjectMetadata_DependsOn>
+        <MicrosoftTye_GetProjectMetadata_DependsOn Condition="'$(IsCrossTargetingBuild)' == 'true'">Restore</MicrosoftTye_GetProjectMetadata_DependsOn>
```

Comment:
For reference, the other targets such as `GetAssumblyAttributes` are not available for a cross-targeting build (i.e. the outer build) that occurs during the first evaluation of a multi-targeted project.

Reply:
I have to be honest, I have no idea if I have something to do / change with that statement now ^^

Reply:
No worries, I just included some comments in case you were wondering what this is for.